index.d.ts

```diff
@@ -13,29 +13,19 @@ interface LatLon {
     lon: number;
 }
 
-/**
- * Return the Haversine distance in meters
- * @param a - first location
- * @param b - second location
- */
-declare function haversineDistance(a: LatitudeLongitude, b: LatitudeLongitude): number;
+interface GeoJSONPoint extends Array<number|number>{0:number; 1:number}
 
-/**
- * Return the Haversine distance in meters
- * @param a - first location
- * @param b - second location
- */
-declare function haversineDistance(a: LatLng, b: LatLng): number;
```

Comment:
Although these were nice,  they don't reflect the implementation.  :+1:

Reply:
That's why I cleaned up a bit 😄👍
Thank you for the merge